test/shopify-cli/tasks/select_org_and_shop_test.rb

```diff
@@ -3,40 +3,27 @@
 module ShopifyCli
   module Tasks
     class SelectOrgAndShopTest < MiniTest::Test
-      include TestHelpers::Partners
```

Comment:
Just double checking: have you checked whether this file needs to be kept after these changes?

Reply:
It does, there's a bunch of other tests that also use `stub_partner_req`.